infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackController.kt

```diff
@@ -208,7 +209,7 @@ class LocationTrackController(
     }
 
     @PreAuthorize(AUTH_ALL_READ)
-    @GetMapping("/{publicationState}/by-tracknumber/{trackNumberId}")
+    @GetMapping("/track-numbers/{publicationState}/location-tracks/{trackNumberId}")
```

Comment:
Tämä urli voisi mielestäni olla muiden `/track-numbers` alla olevien tapaan `.../{publicationState}/{trackNumberId}/location-tracks`, muuten jepa

Reply:
Totta, tuohan on ihan hassusti-päin.